README.md

```diff
@@ -73,9 +153,42 @@ The deflate and zlib specifications were written by L. Peter Deutsch.
 zlib was originally created by Jean-loup Gailly (compression)
 and Mark Adler (decompression).
 
-
-Build Status
-------------
-
-Travis CI: [![build status](https://api.travis-ci.org/zlib-ng/zlib-ng.svg)](https://travis-ci.org/zlib-ng/zlib-ng/)
-Buildkite: [![Build status](https://badge.buildkite.com/7bb1ef84356d3baee26202706cc053ee1de871c0c712b65d26.svg?branch=develop)](https://buildkite.com/circlestorm-productions/zlib-ng)
+Contents
+--------
+
+| Name             | Description                                                    |
+|:-----------------|:---------------------------------------------------------------|
+| arch/            | Architecture-specific code                                     |
+| doc/             | Documentation for formats and algorithms                       |
+| test/example.c   | Zlib usages examples for build testing                         |
+| test/minigzip.c  | Minimal gzip-like functionality for build testing              |
+| test/infcover.c  | inf*.c code coverage for build coverage testing                |
+| win32/           | Shared library version resources for Windows                   |
+| CMakeLists.txt   | Cmake build script                                             |
+| configure        | Bash configure/build script                                    |
+| adler32.c        | Compute the Adler-32 checksum of a data stream                 |
+| compress.c       | Compress a memory buffer                                       |
+| deflate.*        | Compress data using the deflation algorithm                    |
+| deflate_fast.c   | Compress data using the fast strategy of deflation algorithm   |
+| deflate_medium.c | Compress data using the medium stragety of deflation algorithm |
+| deflate_slow.c   | Compress data using the slow strategy of deflation algorithm   |
+| functable.*      | Struct containing function pointers to optimized functions     |
+| gzclose.c        | Zlib function for closing gzip files                           |
+| gzendian.h       | BYTE_ORDER for endian tests                                    |
+| gzguts.h         | Zlib header internal definitions for gz* operations            |
+| gzlib.c          | Zlib functions common to reading and writing gzip files        |
+| gzread.c         | Zlib functions for reading gzip files                          |
+| gzwrite.c        | Zlib functions for writing gzip files                          |
+| infback.*        | Inflate using a callback interface                             |
+| inflate.*        | Zlib decompression                                             |
+| inffast.*        | Fast decoding                                                  |
```

Comment:
Fast decoding of what...

Reply:
Some of these descriptions were taken from the header/source files. I will make some edits, but if you have a suggestion of what to put I can change it.